src/test/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticatorTest.java

```diff
@@ -371,12 +371,12 @@ public void rolesTest() throws Exception {
         mockSamlIdpServer.setSignResponses(true);
         mockSamlIdpServer.loadSigningKeys("saml/kirk-keystore.jks", "kirk");
         mockSamlIdpServer.setAuthenticateUser("horst");
-        mockSamlIdpServer.setAuthenticateUserRoles(Arrays.asList("a", "b"));
+        mockSamlIdpServer.setAuthenticateUserRoles(Arrays.asList("a ,c", "b   ,d,   e", "f"));
```

Comment:
looks good, add `",,"` and `", ,"` test cases.

Reply:
Done.